### DISCUSSION TITLE
Arch+KDE+Vscode unable to login via Github GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown

### DISCUSSION THREAD

**Discussion by pluveto** [community/community#21067] [Copilot Conversations] [answered]
## Version

Kernel: x86_64 Linux 5.18.12-arch1-1
DE: KDE 5.96.0 / Plasma 5.25.3
VSCode: 1.16.2

## Reproduce

1. Install vscode and the copilot extension
2. Login via Github
3. (Agreen in browser)
4. Vscode report error

```
Writing login information to the keychain failed with error 'GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files'.
```

![image](https://user-images.githubusercontent.com/50045289/180432300-a1d30113-00e6-4c3b-946c-6157875c5c8f.png)

![image](https://user-images.githubusercontent.com/50045289/180432332-049ad3cc-dd83-4e33-ac99-88e2c14bafd2.png)

![image](https://user-images.githubusercontent.com/50045289/180432371-4be07564-5a56-4a0a-bddb-5ddd7f84c127.png)

![image](https://user-images.githubusercontent.com/50045289/180432387-687aadb1-e81c-4576-b9e2-77639962b64b.png)

**Comment by pluveto** [comment, marked as answer]
```
sudo pacman -S gnome-keyring
```
works for me.

**Reply by AlejandroFNadal** [reply]
It also works in Linux if you are using KDE: 

`apt install gnome-keyring`
